resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -29,21 +29,6 @@
             <stringProp name="Argument.value">443</stringProp>
             <stringProp name="Argument.metadata">=</stringProp>
           </elementProp>
-          <elementProp name="relyingPartyId" elementType="Argument">
-            <stringProp name="Argument.name">relyingPartyId</stringProp>
-            <stringProp name="Argument.value">Bharathi-Inc</stringProp>
-            <stringProp name="Argument.metadata">=</stringProp>
```

Comment:
Why "Authorization Code Endpoint API" -> "Store Code To File" - why is this disabled, if not required, please remove the same

Reply:
Removed it.